src/generators/wrappers/nullable.ts

```diff
@@ -12,7 +12,7 @@ export const ${getConfig().nullableName} = <T extends TSchema>(schema: T) => ${
 export function nullableImport() {
   return `import { ${getConfig().nullableName} } from "./${
     getConfig().nullableName
-  }"\n`;
+  }${getConfig().explicitFileExtensions ? ".js" : ""}"\n`;
```

Comment:
This should work if we don't need a customizeable ending. 

Is this change enough? I think we never actually import any modules somewhere else, besides the barrel file. Could you take a look at the barrel file and check real quick if the imports do work in your setup? Maybe that thing needs the adjusted imports too!

Reply:
Oh good catch! I don't use the barrel file so didn't spot that.